GVFS/GVFS.UnitTests/CommandLine/HooksInstallerTests.cs

```diff
@@ -75,7 +78,7 @@ public void MergeHooksDataDiscardBlankLines()
             resultLines.Count().ShouldEqual(3);
             resultLines.ElementAt(0).ShouldEqual("first");
             resultLines.ElementAt(1).ShouldEqual("second");
-            resultLines.ElementAt(2).ShouldEqual(GVFSPlatform.Instance.Constants.GVFSHooksExecutableName);
+            resultLines.ElementAt(2).ShouldContain(GVFSPlatform.Instance.Constants.GVFSHooksExecutableName);

```

Comment:
Same comment for these `ShouldEqual` -> `ShouldContain` changes.  Compute the full path and check for equality with it?